README.md

```diff
@@ -76,6 +76,29 @@ And this is what gets rendered (plus the CSS and Javascript you've specified):
 
 ## Release notes
 
+🚨📢 **Version 0.110**
+- BREAKING CHANGE:
+    - Installation changes:
+        - If your components include JS or CSS, you now must use the middleware and add django-components' URLs to your `urlpatterns`
+            (See "[Adding support for JS and CSS](#adding-support-for-js-and-css)")
+    - `preload` kwarg of `{% component_dependencies %}` tag was removed. This will be replaced with HTML fragment support.
+    - Component typing signature changed from `Component[Args, Kwargs, Data, Slots]` to `Component[Args, Kwargs, Slots, Data, JsData, CssData]`
+    - If you rendered a component A with `.render()` and then inserted that into another component B, now you must pass `kind=None` to component A:
```

Comment:
This is described more in-depth further down in the section [`Inserting pre-rendered HTML into another component`](https://github.com/EmilStenstrom/django-components/pull/688/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R3296).

As I was updating the README, I realized that there could be a way to be able to insert one rendered component into another without requiring the `kind=None`.

But I will look into that in a separate MR, since this one is already too big.